frontend/public/locales/zh/ingress.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Not configured": "未配置",
+  "Host": "主机",
+  "Path": "路径",
+  "Service": "Service",
```

Comment:
"Service" could be translated into Chinese.

Reply:
This has been updated.